arm-wt-22k/lib_src/eas_voicemgt.c

```diff
@@ -307,8 +325,36 @@ EAS_RESULT VMInitialize (S_EAS_DATA *pEASData)
     pSecondarySynth->pfInitialize(pVoiceMgr);
 #endif
 
+#ifdef _CC_CHORUS
+    result = EAS_Chorus.pfInit(pEASData, &pVoiceMgr->chorusData);
```

Comment:
```suggestion
    pEAS_Chorus = pEASData->effectsModules[EAS_MODULE_CHORUS].effect;
    if (pEAS_Chorus) {
        result = pEAS_Chorus->pfInit(pEASData, &pVoiceMgr->chorusData);
```
Etc... Replace `EAS_Chorus.` by `pEAS_Chorus->` in the following lines.

Anyway, `pfInit()` is already being called in `EAS_Init()`  in eas_public.c:#L428-L440 but the initialization order in EAS_Init() should be adjusted. Call `VMInitialize()` after the initialization of the effect modules, and then avoid calling `pfInit()` twice.

https://github.com/pedrolcl/sonivox/blob/7e8546189d8596d42943dcc8cabbc8b4fc958554/arm-wt-22k/lib_src/eas_public.c#L428-L440

This should be moved:

https://github.com/pedrolcl/sonivox/blob/7e8546189d8596d42943dcc8cabbc8b4fc958554/arm-wt-22k/lib_src/eas_public.c#L417-L419